src/Tasks/Microsoft.NET.Build.Tasks/build/platforms/Portable.v4.5.targets

```diff
@@ -0,0 +1,47 @@
+﻿<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+
+  <PropertyGroup Condition="$(_PortableTfmCount) == 2 and                             
```

Comment:
How were all these conditions generated?  Did you construct them manually or write a generator for them based that uses the PCL reference assemblies as input?

Also, what happens if someone specifies a Xamarin target framework or an invalid one in the list?